docs/components/gltf-model.md

```diff
@@ -74,10 +74,11 @@ file.
 
 ## Values
 
-| Type     | Description                          |
-|----------|--------------------------------------|
-| selector | Selector to an `<a-asset-item>`      |
-| string   | `url()`-enclosed path to a glTF file |
+| Type     | Description                                 |
+|----------|---------------------------------------------|
+| selector | Selector to an `<a-asset-item>`             |
+| string   | `url()`-enclosed path to a glTF file        |
+| Data URL | data:application/octet-stream;base64,<data> |
```

Comment:
The official `model/gltf-binary` mime type can be used as well.

Reply:
Yes indeed, let's use that one in the table. The application/octet-stream is a generic mimetype for any binary files. I also added a paragraph how to create a Data URL.